### DISCUSSION TITLE
Include sqlite3 binary in docker image

### DISCUSSION THREAD

**Discussion by nitrotm** [dani-garcia/vaultwarden#4543] [Ideas]
It would be great if sqlite3 was included within the official docker image, so that a backup could be performed within the container, eg. using k8up backup operator:

sqlite3 /data/db.sqlite3 .dump

This shouldn't add much overhead (ie. 1.65MB in alpine) and would ease adding automated backup.

**Comment by BlackDex** [comment]
I'm planning on solving this a different way then adding a binary which might be a different version then used by the Vaultwarden binary.

I only didn't had time yet to add this functionality.

**Comment by nitrotm** [comment]
Ok, something like a cli command included in vaultwarden binary?

**Reply by BlackDex** [reply]
Yes, and/or a user kill signal like usr1 or usr2 for example.

**Reply by nitrotm** [reply]
For my use-case, it would be best if the dump can simply be emitted on stdout (as k8up can then compress/encrypt/store it).
